beacon-chain/BUILD.bazel

```
@@ -93,6 +93,12 @@ go_test(
     deps = ["@com_github_urfave_cli//:go_default_library"],
 )
 
+go_path(
+    name = "beacon-chain-path",
+    deps = [":go_default_library"],
+    mode = "copy",
```

```suggestion
    mode = "copy",
    tags = ["manual"],
```

This prevents this target from building unnecessarily, unless you explicitly ask for it. I think it makes a copy on disk which is wasteful unless you wanted it.

---

Ah good idea, ty.